manuscript.md

```diff
@@ -353,47 +362,79 @@ sampling to generate a posterior estimate of parameters, $P(\theta | \hat{x},
+neural-network to perform binary classification by predicting the
+presence/absence of an interaction between any two species. There are two ways
+for the model to be right: the model predicts an interaction and there is one (a
+_true positive_ (TP)), or the model predicts no interaction and there isn't one (a
+_true negative_ (TN)). Similarly, there are two ways for the model to be wrong: the
+model predicts an interaction which does not exist (a _false positive_ (FP)), or the
+model predicts no interaction but it does exist (a _false negative_ (FN)).
+
+A naïve initial approach to measure how well a model does is _accuracy_ - the
+proportion of values it got correct. However, consider what we know about
+interaction networks: they are often very sparse, with connectance between $0.1$
+and $0.3$.  If we build a model that always guesses there will be no interaction
+between two species, it will be correct in the majority of cases because the
+majority of potential interactions in a network typically do not exist.
+Therefore this "empty-matrix" model would always have an _accuracy_ of $1-C$,
+where $C$ is the observed connectance, which would almost always be greater than
+50%. In response to the general problem of biases in classifiers, many metrics
+have been proposed to measure binary-classifiers (@tbl:table). These can be divided
+into two general categories: those in the precision-recall space and those in
+the sensitivity-specificity space.
+
+
+| Name                         | Description                                  | Computation  |
+|-----------------------------:|---------------------------------------------:|-------------:|
+| Accuracy                     | percentage of correct predictions            |    $\frac{\text{TP}+\text{FP}}{\text{TP}+\text{FP}+\text{TN}+\text{FN}}$ |
+| Sensitivity                  | synonymous with true positive rate or recall |    $\frac{\text{TP}}{\text{TP}+\text{FN}}$ |
+| Specificity                  | synonymous with true negative rate           |    $\frac{\text{TN}}{\text{TN}+\text{FP}}$ |
+| Precision                    | f                                            |    $\frac{\text{TP}}{\text{TP}+\text{FP}}$ |
+| True Skill ($TSS$)           | f                                            |  $\frac{\text{TP}}{\text{TP}+\text{FN}} + \frac{\text{TN}}{\text{TN}+\text{FP}} - 1$ |
+| F-1 score ($F_1$)            | f                                            |    $F_1 =  \frac{\text{TP}}{\text{TP}+ 0.5(\text{FN}+\text{FP})}$ |
+| General F-Score ($F_\alpha$) | f                                            |    $F_\alpha =  \frac{(1+\alpha^2)*\text{TP}}{(1+\alpha^2)*\text{TP}+ \alpha^2*\text{FN}+\text{FP}}$ |
+| Cohen's $\kappa$             | f                                            |    $\kappa =  \frac{2(\text{TP}*\text{TN} - \text{FN}*\text{FP})}{(\text{TP}+\text{FP})(\text{FP}+\text{TN})+(\text{TP}+\text{FN})(\text{FN}+\text{TN})}$ |
+
+Table: Table of shit. {#tbl:table}
```

Comment:
If we do end up going with the table (which at this point I'm not offended by) we _could_ consider dropping the computation section? On the fence though. It's very dense and I'd rather dedicate more resources to telling the reader _what_ we can glean from the different methods as opposed to how to implement them. BUT how we calculate them also informs as to what they say so yeah on the fence

Reply:
table was originally tims suggestion in #127. i could go either way on including the computation column

Reply:
Lets see what Tim says then RE computation